servant-client/test/Servant/SuccessSpec.hs

```diff
@@ -83,6 +83,9 @@ successSpec = beforeAll (startWaiApp server) $ afterAll endWaiApp $ do
       let p = Person "Clara" 42
       left show <$> runClient (getBody p) baseUrl `shouldReturn` Right p
 
+    it "Servant.API.Get redirection" $ \(_, baseUrl) -> do
```

Comment:
The idea of this test being that before your patch, we'd get a `ClientError` because we don't get a 2xx status code?

Reply:
Yes, that's the idea. I also checked the same test in `master` and it returns, as expected, a `FailureResponse`.

Reply:
Btw, this is similar to the `"Redirects when appropriate"` test, that uses `uverbGetSuccessOrRedirect True`.

Reply:
OK, sounds good. Since it wasn't 100% obvious to me initially, that might be the case for other contributors, perhaps indicating the purpose in the test label or a comment nearby would help. But the test otherwise looks good.